README.md

```diff
@@ -1 +1,108 @@
+    - `call_params` a hashref of arguments on top of `req_storage` to send to remote method. This will be suplemented with `$req_storage->{args}`
+    added as an `args` key and be merged with `$req_storage->{stash_params}` with stash\_params overwriting any matching
+    keys in `call_params`.
+    - `rpc_response_callback`  If supplied this will be run with args: `Mojolicious::Controller` instance, the rpc\_response and `$req_storage`.
+    **Note:** if `rpc_response_callback` is supplied the success and error callbacks are not used.
+    - `before_get_rpc_response`  arrayref of subroutines to be run before the remote response is received, with args: `$c` and `req_storage`
+    - `after_get_rpc_response` arrayref of subroutines to be run after the remote response is received; called with args: `$c` and `req_storage`
+    called only when there is an response from the remote call.
+    - `before_call` arrayref of subroutines called before the request to the remote service is made.
+    - `rpc_failure_cb` a subroutine reference to call if the remote call fails. Called with `Mojolicious::Controller`, the rpc\_response and `$req_storage`
+    - `msg_group` - if supplied, the message will be assigned to the Redis channel with the corresponding name. The _general_ channel will be used by default if either `$msg_type` is not provided or `queue_separation_enabled` is 0.
+
+Returns undef.
+
+## request
+
+Sends request to backend service. The method accepts following arguments:
+
+- `request_data` - an `arrayref` containing data for the item which is going to be put into redis stream.
+- `$category_name` - this will be passed to `_send_request` to specify which redis category this message belongs to.
+- `category_timeout` - timeout value for this specific call (differs based on category)
+
+Returns future.
+Which will be marked as done in case getting response from backend server.
+And it'll be marked as failed in case of request timeout or in case of error putting request to redis stream.
+
+## wait\_for\_messages
+
+By using redis subscription, we subscribe on channel for receiving responses from backend server.
+We'll use uniq id generated by [whoami](https://metacpan.org/pod/whoami) as subscription channel.
+Subscription will be done only once within first request to backend server.
+
+# SEE ALSO
+
+[Mojolicious::Plugin::WebSocketProxy](https://metacpan.org/pod/Mojolicious%3A%3APlugin%3A%3AWebSocketProxy),
+[Mojo::WebSocketProxy](https://metacpan.org/pod/Mojo%3A%3AWebSocketProxy)
+[Mojo::WebSocketProxy::Backend](https://metacpan.org/pod/Mojo%3A%3AWebSocketProxy%3A%3ABackend),
+[Mojo::WebSocketProxy::Dispatcher](https://metacpan.org/pod/Mojo%3A%3AWebSocketProxy%3A%3ADispatcher),
+[Mojo::WebSocketProxy::Config](https://metacpan.org/pod/Mojo%3A%3AWebSocketProxy%3A%3AConfig)
+[Mojo::WebSocketProxy::Parser](https://metacpan.org/pod/Mojo%3A%3AWebSocketProxy%3A%3AParser)
+
+# COPYRIGHT AND LICENSE
+
+Copyright (C) 2022 deriv.com
```

Comment:
```suggestion
Copyright (C) 2022 Deriv Group Services Ltd
```

Reply:
@chylli-binary is it okay to commit this suggestion?